addons/rose/addon/components/rose/form/checkbox/group/index.js

```diff
@@ -25,13 +24,51 @@ export default class RoseFormCheckboxGroupComponent extends Component {
     const selectedItems = this.args.selectedItems || [];
     const currentItems = [...selectedItems];
 
-    if (currentItems.includes(item)) {
-      const i = currentItems.indexOf(item);
+    if (this.doesItemExist(currentItems, item)) {
+      const i = this.getItemIndex(currentItems, item);
       currentItems.splice(i, 1);
     } else {
       currentItems.push(item);
     }
 
     if (this.args.onChange) this.args.onChange(currentItems);
   }
+
+  /**
+   * Checks if the item is in the current items list.
+   * If @itemEqualityFunc is not specified, will do a simple `includes` on the list.
+   * @param currentItems
+   * @param item
+   * @returns {boolean}
+   */
+  @action
+  doesItemExist(currentItems, item) {
```

Comment:
Should this be `has` or `includes` or something similar?  The semantics are about inclusion in the list.

Reply:
Since the selected items are an argument to the component already, we shouldn't need to pass them again here.

Reply:
Good point. I updated these in #1335.